theme/json/posts.json

```diff
@@ -1,29 +1,38 @@
 [
 	{
+		"id": "101",
 		"title":"Five Trends in Today's Youth Activism",
 		"body":"Want to know what you can learn from today's youth activism? Here are five trends from the global youth climate movement.",
 		"linkText":"Read More...",
 		"link":"#",
-		"imageLink":"theme/img/volunteer.jpeg",
+		"imageLink":"/theme/img/volunteer.jpeg",
 		"imageAltText":"volunteers",
-        "date": "18 July 2019"
+        "date": "18 July 2019",
+		"authorName": "Olivia Benson",
```

Comment:
Start a new array group called author and add authorName and AuthorImage inside them and you can just call them author and image.

Reply:
done